src/index.js

```diff
@@ -23,11 +22,34 @@ export default function workerize(code, options) {
 	if (typeof code==='function') code = `(${Function.prototype.toString.call(code)})(${exportsObjName})`;
 	code = toCjs(code, exportsObjName, exports) + `\n(${Function.prototype.toString.call(setup)})(self,${exportsObjName},{})`;
 	let url = URL.createObjectURL(new Blob([code])),
-		worker = new Worker(url, options),
-		term = worker.terminate,
 		callbacks = {},
 		counter = 0,
+		worker,
+		term,
 		i;
+
+	try {
+		worker = new Worker(url, options);
+		term = worker.terminate;
+	} catch (err) {
+		// TODO: How to indicate the fallback? console.warn? Property on the export?
+		// TODO: What's the right way to access the original exports?
+		const originalExports = new Function(code + `\nreturn ${exportsObjName}`)();
+		// TODO: Is this the full API? Do we need to add anything else?
+		worker = {
+			kill() {},
+			terminate() {},
```

Comment:
Should we implement these methods to remove the proxied methods on the worker? Or fine as no-op?

Reply:
TBH I'm tempted to not even emulate these since they do nothing. Doing an existence-test for `worker.terminate` would be a decent way to know if the worker got inlined or not.